### PR TITLE
Pr for adding Pop!_Os inspired theme

### PR DESCRIPTION
**Pull request by arthurhawron**
### Description

Closes # I tried to make a theme inspired by system76's linux distro pop os. It can be found here: [https://pop.system76.org](url).
Here are some screenshots of the theme: test_end in_test

### CODE REVIEW

**Review by rizwanmustafa — APPROVED** [review]
As far as the code goes, all is well

File: frontend/static/themes/pop.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    --bg-color: #353432;
+    --main-color: #F8B571;
+    --caret-color: #F8B571;
+    --sub-color: #6C6B69;
+    --sub-alt-color: #ffffff;
+    --text-color: #E6E5E3;
+    --error-color: #83C2CE;
+    --error-extra-color: #ffffff;
+    --colorful-error-color: #ffffff;
+    --colorful-error-extra-color: #ffffff;
```
Comment (rizwanmustafa) [inline frontend/static/themes/pop.css:11]: Have a different color than `--colorful-error-color`

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]
The theme color needs some changes

**Review by arthurhawron — COMMENTED** [review]
Changed --colorful-error-color to a light blue

**Comment by rizwanmustafa** [comment]
> Changed

Great. Now the final theme approval will be done by Miodec (Project Owner).

**Comment by Miodec** [comment]
It looks like a desaturated version of carbon - bit too similar, there are already too many themes on the website. Sorry.

**Pull request closed without merging**